book/translation.rst

```diff
@@ -502,6 +502,11 @@ as the locale for the current request.
 You can now use the locale to create routes to other translated pages
 in your application.
 
+.. tip::
+
+    Read :doc:`/cookbook/routing/service_container_parameters` to learn, how to
+    avoid hardcoding the ``_locale`` requirement in all your routes.
```

Comment:
I have a question about the use of the word `requirement` in this tip. Are you referring to a real route `requirement` option or are you using it as a regular word? If that's the case, we could change it to avoid confusions.